.travis.yml

```diff
@@ -20,11 +24,14 @@ matrix:
     - env: GCC=gcc-6     CXX=g++-6       LIBTYPE=system  CMAKE_OPT="-DWITH_READLINE=system" PACKAGES=g++-6     PPA=ubuntu-toolchain-r/test
     - env: GCC=clang-4.0 CXX=clang++-4.0 LIBTYPE=system  CMAKE_OPT="-DWITH_READLINE=system" PACKAGES="clang-4.0 llvm-4.0-dev" PPA=ubuntu-toolchain-r/test LLVM=llvm-toolchain-trusty-4.0
     - env: GCC=clang-5.0 CXX=clang++-5.0 LIBTYPE=bundled CMAKE_OPT="-DWITH_PAM=1"           PACKAGES="clang-5.0 llvm-5.0-dev" LLVM=llvm-toolchain-trusty-5.0
+    - os: osx
+      osx_image: xcode9.3
+      env: LIBTYPE=bundled GCC=clang CXX=clang++ MAINTAINER_MODE=OFF WITHOUT_TOKUDB=ON
```

Comment:
Difference from 5.5 review comments: the OpenSSL system CMake option becomes -DWITH_SSL=/usr/local/open/openssl/, and also possible to do brew install libevent and -DWITH_LIBEVENT=system

Reply:
As I mentioned for the 5.5 pr: I didn't see any consistent speedup, because brew also takes time. Also, the added option would make the travis config more complex.

Otherwise I updated the PR based on the changes in the 5.5 version.